tools/openblas_support.py

```diff
@@ -54,39 +55,67 @@ def get_ilp64():
 
 
 def get_manylinux(arch):
-    if arch in ('x86_64', 'i686'):
+    if arch in ('i686'):
         default = '2010'
     else:
         default = '2014'
-    ret = os.environ.get("MB_ML_VER", default)
+    ml_ver = os.environ.get("MB_ML_VER", default)
     # XXX For PEP 600 this can be a glibc version
-    assert ret in ('1', '2010', '2014', '_2_24'), f'invalid MB_ML_VER {ret}'
-    return ret
+    assert ml_ver in ('2010', '2014', '_2_24'), f'invalid MB_ML_VER {ml_ver}'
+    suffix = f'manylinux{ml_ver}_{arch}.tar.gz'
+    return suffix
```

Comment:
This might be a question for multibuild first, but we should eventually add `_2_28` here, as `_2_24` is EOL and has effectively been replaced by `_2_28`.

Reply:
At the moment the only `ml_ver` options on https://anaconda.org/multibuild-wheels-staging/openblas-libs/files are `{'2010', '2014'}`

Reply:
Multibuild supports `_2_28`. As for openblas-libs, I could build `_2_28` tarballs, but as 2014 will only go EOL in June 2024, I would prefer not to right now. As far as I know, there are no glibc dependencies in OpenBLAS, so there is no compelling reason to update.

I think I started building a `_2_24` openblas-libs tarball at one point but reached the conclusion that it was not needed, so line 64 could be limited to the versions currently on anaconda.org: 2010 and 2014 (and maybe even just 2014 alone, I may drop 2010 soon).